package.json

```diff
@@ -130,5 +130,8 @@
   "workspaces": [
     "modules/**"
   ],
-  "browserslist": "> 5%, last 2 firefox versions, last 2 chrome versions, last 2 safari versions, last 2 edge versions, ie 11"
+  "browserslist": "> 5%, last 2 firefox versions, last 2 chrome versions, last 2 safari versions, last 2 edge versions, ie 11",
+  "dependencies": {
```

Comment:
This changeset is odd, did you add `commitlint` to dependencies and then remove?

Reply:
Agreed. I can fix this and merge today